plugins/module_utils/storage/dell/shared_library/writable_snapshot.py

```diff
@@ -0,0 +1,58 @@
+# Copyright: (c) 2024, Dell Technologies
```

Comment:
I guess this file is not required; we can add these methods into existing snapshot file as it comes under snapshot_api as well.

Reply:
done